packages/composites/price/src/endpoint/price/index.ts

```diff
@@ -0,0 +1,67 @@
+import { Validator } from '@chainlink/ea-bootstrap'
+import {
+  AdapterContext,
+  AdapterRequest,
+  AdapterResponse,
+  ExecuteWithConfig,
+  InputParameters,
+} from '@chainlink/types'
+import { Config } from '../../config'
+import { convertUSDQuote, getTokenPrice } from '../../utils'
+import * as beth from './beth'
+
+export const supportedEndpoints = ['price']
+
+export const inputParameters: InputParameters = {
+  from: ['base', 'from', 'coin'],
+  to: ['quote', 'to', 'market'],
+  quoteDecimals: false,
+  source: false,
+}
+
+export type PriceExecute = (
+  input: AdapterRequest,
+  context: AdapterContext,
+  config: Config,
+  taAdapterResponse: AdapterResponse,
+) => Promise<AdapterResponse>
+
+export const execute: ExecuteWithConfig<Config> = async (input, context, config) => {
+  const validator = new Validator(input, inputParameters)
+  if (validator.error) throw validator.error
+
+  const { from, to, quoteDecimals } = validator.validated.data
+  const fromUpperCase = from.toUpperCase()
+
+  let taDecimals: number
+  let priceExecute: PriceExecute
+  let intermediaryTokenSymbol: string
+  switch (fromUpperCase) {
+    case beth.FROM:
+      taDecimals = beth.INTERMEDIARY_TOKEN_DECIMALS
```

Comment:
I go back and forth between hardcoding this and passing it as a request param.  I decided to hardcode it so that the caller doesn't have to think about it.  Happy to get other opinions.

Reply:
Since the price endpoint doesn't require a decimals param on any other adapter, I'm good with hard-coding it. The job naturally handles the required amount of decimals to go to a feed anyway, we only need to be sure that we have enough precision.